lib/pages/home/favourite_websites_card.dart

```diff
@@ -25,6 +25,7 @@ class FavouriteWebsitesCard extends StatelessWidget {
       title: S.current.sectionFrequentlyAccessedWebsites,
       onShowMore: onShowMore,
       future: websiteProvider.fetchFavouriteWebsites(uid),
+      showIfEmpty: false,
```

Comment:
This is now the default, no need to specify it. Remove it from both files please.

Reply:
Removed it, just to mentione, I opened a new PR, this time not from a fork, so that all automatic tests would run as expected